src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -2029,6 +2030,46 @@ public void validateDeleteRequest(
     return rowCounts;
   }
 
+  private static final String exportToParquetTemplate =
+      "export data OPTIONS( "
+          + "uri='<exportPath><table>-*.parquet', "
+          + "format='PARQUET') "
+          + "AS select * from `<project>.<snapshot>.<table>`";
+
+  private static final String exportPathTemplate = "gs://<bucket>/<flightId>/<table>/";
+
+  public List<String> exportTableToParquet(
+      Snapshot snapshot, GoogleBucketResource bucketResource, String flightId)
+      throws InterruptedException {
+    List<String> paths = new ArrayList<>();
+    String snapshotProject = snapshot.getProjectResource().getGoogleProjectId();
+    String snapshotName = snapshot.getName();
+    BigQueryProject bigQueryProject = BigQueryProject.from(snapshot);
+
+    for (var table : snapshot.getTables()) {
+      String tableName = table.getName();
+      String exportPath =
+          new ST(exportPathTemplate)
+              .add("bucket", bucketResource.getName())
+              .add("flightId", flightId)
+              .add("table", tableName)
+              .render();
+
+      String exportStatement =
+          new ST(exportToParquetTemplate)
+              .add("exportPath", exportPath)
+              .add("table", tableName)
+              .add("project", snapshotProject)
+              .add("snapshot", snapshotName)
+              .render();
+
```

Comment:
In synapse, we have to drop the intermediate table created by the parquet create. Is this not needed here?

Reply:
Oh wait, there is no intermediate table, right?